lxd/instances_get.go

```diff
@@ -381,14 +386,15 @@ func doInstancesGet(d *Daemon, r *http.Request) (interface{}, error) {
 		// containers from their respective nodes.
 		if mustLoadObjects && address != "" && !isClusterNotification(r) {
 			wg.Add(1)
-			go func(address string, containers []string) {
+			go func(address string, instancesAndProjects [][2]string) {
 				defer wg.Done()
 
 				if recursion == 1 {
 					cs, err := doContainersGetFromNode(filteredProjects, address, allProjects, networkCert, d.serverCert(), r, instanceType)
 					if err != nil {
-						for _, name := range containers {
-							resultListAppend(name, api.Instance{}, err)
+						for _, instanceAndProjectNames := range instancesAndProjects {
+							instanceName := instanceAndProjectNames[0]
+							resultListAppend(instanceName, api.Instance{}, err)
```

Comment:
These two functions `resultListAppend` and `resultFullListAppend` look suspicious in that they take only an instanceName argument.

Reply:
The name is only used. I will pass in the project as well, if anything that will improve the error info.

Reply:
Yes, thanks.